docs/api-usage.rst

```diff
@@ -411,6 +411,27 @@ user. For example:
 
    p = gl.projects.create({'name': 'awesome_project'}, sudo='user1')
 
+.. warning:: 
+   When using ``sudo``, its usage is not remembered. What this means is that if
+   you use ``sudo`` to retrieve an object and then later use ``save()`` to
+   modify the object it will not use ``sudo``. In that case you should do
+   ``save(sudo='user1')``
```

Comment:
Could we maybe shorten the sentence a bit as it's inside an admonition block, so we can also mention why this is needed?

```suggestion
   When using ``sudo``, its usage is not remembered. If you use ``sudo`` to retrieve an object
   and then later use ``save()`` to modify the object, it will not use ``sudo``.  You should use
   ``save(sudo='user1')`` if you want to perform subsequent actions as the  user.
```

Reply:
Done.